lib/awful/client.lua

```diff
@@ -900,19 +900,18 @@ function client.setwfact(wfact, c)
     t:emit_signal("property::windowfact")
 end
 
---- Increment a client's window factor
+--- Change a client's window factor.
 --
 -- @legacylayout awful.client.incwfact
--- @param add amount to increase the client's window
+-- @tparam number add Amount to increase/decrease the client's window.
+--   Should be between 0.0 and 1.0.
```

Comment:
@psychon 
Thanks for explaining it.  After all, I consider this pattern / range a bit weird.  It seems to be the cause for incrementing and then decrementing by the same value (e.g. `0.1`, using a key binding) will result in a different size than before.
Could this be fixed somehow to be more "predictable"?  /cc @Elv13